src/Symfony/Component/Form/ClearableErrorInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form;
+
+/**
+ * A form element whose errors can be cleared.
+ *
+ * @author Colin O'Dell <colinodell@gmail.com>
+ */
+interface ClearableErrorInterface
+{
+    /**
+     * Removes all the errors of this form.
+     *
+     * @param bool $deep whether to remove errors from child forms as well
+     *
+     * @return $this
+     */
+    public function clearErrors($deep = false);
```

Comment:
Could `true` be a better default here? How frequent is to clear errors for root form only? I think most of the time it happens inside a child field.

There is an issue about the same default but `getErrors($deep = false)` method https://github.com/symfony/symfony/issues/25738 and was reported as confusing.

Reply:
I don't have a strong preference either way, but I do think the default value here should match `getErrors()`.